### PR TITLE
Adding webview logic for non applicable vulnerabilities.

### PR DESCRIPTION
**Pull request by dortam888**
Did the following fix in Applicability:

Changed title from Applicability evidence to Contextual Analysis.
Created logic for Non-Applicable CVEs to show reason why CVE is not applicable.
Fixed some minor style issues in Applicable text (indent bullets).

Applicable_new_view
Non-Applicable_new-view

Screenshot 2024-10-07 at 0 59 54
Screenshot 2024-10-07 at 10 47 35
Screenshot 2024-10-07 at 10 47 48

linked to vscode fix: https://github.com/jfrog/jfrog-vscode-extension/pull/497

### CODE REVIEW

File: src/components/Page/Dependency/ApplicabilityEvidence.tsx [hunk 1]
```diff
@@ -37,6 +38,21 @@ export default function ApplicabilityEvidence(props: Props): JSX.Element {
...
 					</>
+				) : (
+					// If CVE is not applicable
+					<>
+						<h6 className={css.subtitle}>Why is this CVE not applicable?</h6>
+						<div>
+							<div className={css.rowList}>
+								{props.data.evidence?.map((evidence, i) => (
+									<div key={i}>
+										<Row title="Reason" data={evidence.reason} />
```
Comment (attiasas) [inline src/components/Page/Dependency/ApplicabilityEvidence.tsx:49]: Are we only showing reasons when not applicable?

**Review by attiasas — COMMENTED** [review]
Nice Work!, check out my comments and please add tests.
Make sure `Product` approves UI changes before merge

File: src/components/Page/Dependency/ApplicabilityEvidence.tsx (hunk 1 above)
Comment (dortam888) [reply src/components/Page/Dependency/ApplicabilityEvidence.tsx:49]: yes. See https://jfrog-int.atlassian.net/browse/XRAY-81967 image for non-applicable

File: src/components/Page/Dependency/Dependency.tsx [hunk 2]
```diff
@@ -2,23 +2,22 @@ import css from './Dependency.module.css'
...
-			)}
 			<InformationTabs
 				data={props.data}
-				tabs={[TABS.WHAT_CAN_I_DO.key, TABS.CVE_INFORMATION.key, TABS.IMPACT_GRAPH.key]}
+				tabs={[
+					TABS.WHAT_CAN_I_DO.key,
+					TABS.CVE_INFORMATION.key,
+					TABS.IMPACT_GRAPH.key,
+					TABS.CONTEXTUAL_ANALYSIS.key
+				]}
```
Comment (attiasas) [inline src/components/Page/Dependency/Dependency.tsx:20]: what if not JAS scan (i.e not entitled), should we also show the new tab?

File: src/components/UI/InformationTabs/ApplicabilityEvidence.tsx [hunk 3]
```diff
@@ -37,6 +38,21 @@ export default function ApplicabilityEvidence(props: Props): JSX.Element {
 						</div>
 						<Divider />
 					</>
+				) : (
+					// If CVE is not applicable
```
Comment (attiasas) [inline src/components/UI/InformationTabs/ApplicabilityEvidence.tsx:42]: what if not entitled? or other status?

File: src/components/Page/Dependency/Dependency.tsx (hunk 2 above)
Comment (dortam888) [reply src/components/Page/Dependency/Dependency.tsx:20]: I don't want to add JAS logic in the front so I'll let the back do the logic. 
Added:
	const showContextualAnalysisTab =
		showApplicabilityEvidence && props.tabs.includes(TABS.CONTEXTUAL_ANALYSIS.key)
If not entitled, show applicability will be false and we won't show the tab

File: src/components/UI/InformationTabs/ApplicabilityEvidence.tsx (hunk 3 above)
Comment (dortam888) [reply src/components/UI/InformationTabs/ApplicabilityEvidence.tsx:42]: Currently, there is no logic for other status in the Applicability. I added the status for Applicable and not Applicable in VScode.
When I add the status I'll know what to add in the front.

**Pull request merged**